style.css

```diff
@@ -333,9 +333,14 @@ footer {
 #toc {
 	position: fixed;
 	left: 1%;
+	bottom: 1%;
 	max-width: calc(48% - 450px);
 	font-size: 80%;
-	opacity: .3;
+    z-index: 999;
+    background: rgb(255, 255, 255);
```

Comment:
So, `white`?

Reply:
Sure, whatever style of color suits you. 😉